tokens/mnft.json

```diff
@@ -0,0 +1,10 @@
+{
+    "ethereum_address": "0x39fc9e94caeacb435842fadedecb783589f50f5f",
+    "aurora_address": "",
```

Comment:
Your address in Aurora is: `0xd126b48c072f4668e944a8895bc74044d5f2e85b`. Please add to the PR